extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -627,6 +630,14 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                 return [_errorHookCnt, _autoExceptionInstrumented];
             };
             
+            function _resetAjaxAttempts() {
+                // Reset ajax attempts counter for the new page view
+                let ajaxPlugin = _self.core.getPlugin<IDependenciesPlugin>("AjaxDependencyPlugin");
```

Comment:
Add a check for whether _self.core is defined, so NOT use the "?." operator, keep the generated JavaScript small from an ES5 perspective.